spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -49,12 +49,14 @@
  * @author João André Martins
  * @author Daniel Zou
  */
-public class GoogleStorageResource extends GoogleStorageLocation implements WritableResource {
+public class GoogleStorageResource implements WritableResource {
 
 	private static final Log LOGGER = LogFactory.getLog(GoogleStorageResource.class);
 
 	private final Storage storage;
 
+	private final GoogleStorageLocation location;
```

Comment:
I think it actually makes sense to add a getter for this now.

Reply:
Done.